src/components/Profile/components/PersonalInfoList/index.jsx

```diff
@@ -442,6 +442,7 @@ const PersonalInfoList = ({ myProf, profile, isOnline, createSnackbar }) => {
                 <GordonDialogBox
                   open={isJoinDialogOpen}
                   title={`Mailbox Instructions`}
+                  onClose={() => setIsJoinDialogOpen(false)}
                   closeButtonClicked={() => setIsJoinDialogOpen(false)}
```

Comment:
Could the mailbox instructions dialog use the `cancelButtonClicked` instead, and just pass a `cancelButtonName` of `"Close"`?

```suggestion
                  onClose={() => setIsJoinDialogOpen(false)}
                  cancelButtonClicked={() => setIsJoinDialogOpen(false)}
                  cancelButtonName="Close"
```

Reply:
This works well as well, I'll commit the changes and remove the close button functionality from GordonDialogBox.